src/app/constants/styles.ts

```diff
@@ -0,0 +1,10 @@
+export const assetStyles = {
+    inactive: {
```

Comment:
should this be called `default`?

there is `active`, then custom style, or the default style.

Not really certain though on this one.